index.js

```diff
@@ -245,27 +245,27 @@ async function main (argv) {
       process.exit(1)
     }
 
-    process.stdout.write('Ok.')
+    process.stdout.write('Ok.\n')
 
     try {
-      process.stdout.write('\nSetting up TS configuration...')
+      process.stdout.write('Setting up TS configuration...')
```

Comment:
Just checking, this one doesn't need a trailing `\n`?

Reply:
It doesn't need.
```
Setting up TS configuration...Ok.
```